lite/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM alpine:3.15
+ENV LANG C.UTF-8
+
+RUN apk update
+RUN apk upgrade
+RUN apk add \
+        clang \
+        clang-extra-tools \
+        clang-analyzer \
+        lld \
+        cmake \
+        make \
+        doxygen \
+        git \
+        python3 \
+        py3-pip \
+        bash \
+        gcc-arm-none-eabi \
+        newlib-arm-none-eabi \
+        protoc
+
+# So that it still supports things incorrectly pointing to python
+RUN ln -s python3 $(dirname $(which python3))/python
+
+# These packages contain shared libraries which will be needed at runtime
+RUN apk add \
+        libjpeg \
+        zlib \
+        eudev \
+        libusb
+
+# Python packages building dependencies, can be removed afterwards
+ARG PYTHON_BUILD_DEPS=python3-dev,musl-dev,libusb-dev,eudev-dev,linux-headers,zlib-dev,jpeg-dev
```

Comment:
Same here

Reply:
Done in [fe172af](https://github.com/LedgerHQ/ledger-app-builder/pull/36/commits/fe172afb8eadcb3519017aec39cf3e8784bb0101)